mpd/duration.go

```diff
@@ -54,29 +54,16 @@ func (d *Duration) String() string {
 	if u < uint64(time.Second) {
```

Comment:
Should this be LTE, rather than strictly LT, given the line 59 handles the strictly-equal case?